sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

```diff
@@ -289,6 +289,12 @@ object ShimLoader extends Logging {
     shimProviderClass = classname
   }
 
+  def newClassOf(className: String): Class[_] = {
```

Comment:
nit: consider `loadClass` as the name

nit be made resusable for newInstanceOf